priorityclass/priorityclass.go

```diff
@@ -0,0 +1,59 @@
+package priorityclass
+
+import (
+	"context"
+	"golang.org/x/xerrors"
+	v1 "k8s.io/api/scheduling/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	schedulingv1 "k8s.io/client-go/applyconfigurations/scheduling/v1"
+	clientset "k8s.io/client-go/kubernetes"
+)
+
+// Service manages priorityClass.
+type Service struct {
+	client clientset.Interface
+}
+
+// NewPriorityClassService initializes Service.
+func NewPriorityClassService(client clientset.Interface) *Service {
+	return &Service{
+		client: client,
+	}
+}
+
+func (s Service) Get(ctx context.Context, name string) (*v1.PriorityClass, error) {
```

Comment:
Please fix all receiver to pointer like other service.

```suggestion
func (s *Service) Get(ctx context.Context, name string) (*v1.PriorityClass, error) {
```

Reply:
My mistake, I've changed the receiver from struct to struct pointer